app/views/handlers/feedback.py

```diff
@@ -174,3 +201,47 @@ def is_limit_reached(session_data: SessionData) -> bool:
     @staticmethod
     def is_enabled(schema: QuestionnaireSchema) -> bool:
         return schema.get_submission().get("feedback")
+
+
+class FeedbackMetadata:
+    def __init__(self, feedback_count, form_type, language_code, region_code, tx_id):
+        self.feedback_count = feedback_count
+        self.form_type = form_type
+        self.language_code = language_code
+        self.region_code = region_code
+        self.tx_id = tx_id
+
+    @property
+    def metadata(self) -> Mapping:
```

Comment:
Is this  a pattern used elsewhere? I'd expect a `FeedbackMetadata` class or similar metadata class to return a representation of itself in this format using it's `__call__` method?